packages/apollo-voyager-tools/package.json

```diff
@@ -21,6 +21,9 @@
     "graphql-tools": "^4.0.3"
   },
   "devDependencies": {
+    "@aerogear/apollo-voyager-audit": "0.2.7-rc3",
```

Comment:
Just a minor thing: make sure this and the metrics ones are removed.

Reply:
ahh, never saved the file. 2 secs